Jenkinsfile

```diff
@@ -25,30 +25,40 @@ pipeline {
         }
     }
 
-    stage('publish images on push to master') {
+    stage('publish images') {
         when {
-            branch 'master'
+            expression {!env.BRANCH_NAME.startsWith('PR')}
         }
-        steps {
+        steps{
             withCredentials([usernamePassword(credentialsId: 'ocpmetal_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
                 sh '''docker login quay.io -u $USER -p $PASS'''
             }
-            sh '''docker tag  ${INSTALLER} ${INSTALLER}:latest'''
             sh '''docker tag  ${INSTALLER} ${INSTALLER}:${GIT_COMMIT}'''
-            sh '''docker push ${INSTALLER}:latest'''
             sh '''docker push ${INSTALLER}:${GIT_COMMIT}'''
 
-            sh '''docker tag  ${CONTROLLER} ${CONTROLLER}:latest'''
             sh '''docker tag  ${CONTROLLER} ${CONTROLLER}:${GIT_COMMIT}'''
-            sh '''docker push ${CONTROLLER}:latest'''
             sh '''docker push ${CONTROLLER}:${GIT_COMMIT}'''
 
-            sh '''docker tag  ${CONTROLLER_OCP} ${CONTROLLER_OCP}:latest'''
             sh '''docker tag  ${CONTROLLER_OCP} ${CONTROLLER_OCP}:${GIT_COMMIT}'''
-            sh '''docker push ${CONTROLLER_OCP}:latest'''
             sh '''docker push ${CONTROLLER_OCP}:${GIT_COMMIT}'''
         }
     }
+
+    stage('publish images on push to master') {
+        when {
+            branch 'master'
+        }
+        steps {
+            sh '''docker tag  ${INSTALLER} ${INSTALLER}:latest'''
```

Comment:
you don't need 
withCredentials([usernamePassword(credentialsId: 'ocpmetal_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
                sh '''docker login quay.io -u $USER -p $PASS'''
?
same as in line 33

Reply:
ma?